content/hardware/04.pro/boards/portenta-x8/tutorials/out-of-the-box/content.md

```diff
@@ -59,16 +58,17 @@ Once it is connected, you should see the Wi-Fi status bullet in the bottom left
 
 ***You can change your network by clicking on the button again and repeat the above steps***
 
-## Connect to a FoundriesFactory
+## Connect to a Foundries Factory
```

Comment:
@vanmaegima For consistency should we use it all together? in which cases can we use it with the space? 🤔

Reply:
Let's ask @kprosise about this

Reply:
Clear! We will have this in mind. 
Thanks 😄